src/components/pages/MentorMenteeMatching/MentorOptions.js

```diff
@@ -0,0 +1,18 @@
+import { Select } from 'antd';
+
+export const options = data => {
+  const { Option } = Select;
+
+  return (
+    <>
+      <Option value="Assign Mentor">Assign Mentor</Option>
+      <Option value={data.Mentors.first}>{data.Mentors.first}</Option>
+      <Option value={data.Mentors.second}>{data.Mentors.second}</Option>
+      <Option value={data.Mentors.third}>{data.Mentors.third}</Option>
+    </>
+  );
```

Comment:
Great job refactoring this options component! It looks really clean and simple now.